core/src/main/java/de/danoeh/antennapod/core/service/download/EpisodeDownloadWorker.java

```diff
@@ -262,7 +262,7 @@ private Notification generateProgressNotification() {
             progressCopy = new HashMap<>(notificationProgress);
         }
         for (Map.Entry<String, Integer> entry : progressCopy.entrySet()) {
-            bigTextB.append(String.format(Locale.getDefault(), "%s (%d%%)\n", entry.getKey(), entry.getValue()));
+            bigTextB.append(String.format(Locale.getDefault(), "%s (%d%%)%n", entry.getKey(), entry.getValue()));
```

Comment:
This feels a bit counter intuitive (especially since this code will probably never be run on Windows). Maybe disable this check?

Reply:
Sure.